backend/src/main/java/org/example/backend/model/FriendRequest.java

```diff
@@ -0,0 +1,17 @@
+package org.example.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FriendRequest {
+    private String id;
+    private User sender;
+    private User receiver;
+    private String status;
```

Comment:
Maybe use an Enum for the status? What are the possible states? OPEN, ACCEPTED, DECLINED?

Reply:
Thats a good idea. I changed the String status do a enum like this: 

```
package org.example.backend.model;

import lombok.Getter;

@Getter
public enum Status {
    PENDING("pending"),
    ACCEPTED("accepted"),
    REJECTED("rejected");

    private final String statusValue;

    Status(String statusValue) {
        this.statusValue = statusValue;
    }

}
```